### PR TITLE
Update oidc-auth.md with some format and words change

### PR DESCRIPTION
**Pull request by jessehu**

### CODE REVIEW

File: docs/1.10/administration/configure-authentication/oidc-auth.md [hunk 1]
```diff
@@ -5,26 +5,29 @@ weight: 25
 
 If you select OpenID Connect (OIDC) authentication, users log in to the Harbor interface via an OIDC single sign-on (SSO) provider, such as Okta, KeyCloak, or dex. In this case, you do not create user accounts in Harbor.
 
-{{< important >}}
+---
+**Notice**
```
Comment (stuclem) [inline docs/1.10/administration/configure-authentication/oidc-auth.md]: **Note**

File: docs/1.10/administration/configure-authentication/oidc-auth.md [hunk 2]
```diff
@@ -5,26 +5,29 @@ weight: 25
...
+- To manage users by using OIDC groups, create a custom group claim which contains all of the user groups that you want to register in Harbor. The group claim must be mapped in the ID token which is sent to Harbor when users log in. You can enable the `memberof` feature on the OIDC provider. With the `memberof` feature, the OIDC user entity's `memberof` attribute is updated when the group entity's `member` attribute is updated, for example by adding or removing an OIDC user from the OIDC group.
 - Register Harbor as a client application with the OIDC provider. Associate Harbor's callback URI to the client application as a `redirectURI`. This is the address to which the OIDC provider sends ID tokens.
 
 ### Configure an OIDC Provider in Harbor
 
 Before configuring an OIDC provider in Harbor, make sure that your provider is configured correctly according to the preceding section.
 
-1. Log in to the Harbor interface with an account that has Harbor system administrator privileges.
+1. Log in to the Harbor interface as an account with Harbor system administrator privilege.
```
Comment (stuclem) [inline docs/1.10/administration/configure-authentication/oidc-auth.md]: Log in to the Harbor interface with an account that has Harbor system administrator privileges.

File: docs/1.10/administration/configure-authentication/oidc-auth.md [hunk 3]
```diff
@@ -92,6 +95,10 @@ You can now use your CLI secret as the password when logging in to Harbor from t
 docker login -u testuser -p <i>cli_secret</i> jt-test.local.goharbor.io
 </pre>
 
-{{< note >}}
-The CLI secret is associated with the OIDC ID token. Harbor will try to refresh the token, so the CLI secret will be valid after the ID token expires. However, if the OIDC Provider does not provide a refresh token or the refresh fails, the CLI secret becomes invalid. In this case, log out and log back in to Harbor via your OIDC provider so that Harbor can get a new ID token. The CLI secret will then work again.
-{{< /note >}}
+---
+**Notice**
```
Comment (stuclem) [inline docs/1.10/administration/configure-authentication/oidc-auth.md]: **Note**

**Comment by reasonerjt** [comment]
Closing as the Doc has been moved to antoher repo.

**Pull request closed without merging**